### DISCUSSION TITLE
Issues with GitHub Pages Deployment

### DISCUSSION THREAD

**Discussion by Debanjan110d** [community/community#136294] [Actions] [answered]
### Select Topic Area

Show & Tell

### Body

Hello GitHub Community,

I'm encountering an issue when trying to deploy my site using GitHub Pages. Every time I attempt to upload something, I receive a security warning indicating that the site might be dangerous and could potentially deceive users into installing software or disclosing sensitive information.

Here are the steps I have already taken to try and resolve the issue:

1. Checked for Malware or Phishing Content: I reviewed all the HTML, CSS, JavaScript, and other files to ensure they are clean and do not contain any harmful code.
2. Verified HTTPS Configuration: I made sure that my site is configured to use HTTPS.
3. Reviewed GitHub Pages Settings: I checked the settings in my repository to ensure the source branch and folder are correctly set and that there are no warnings or errors.
4. Cleared Browser Cache and Cookies: I cleared my browser cache and cookies to rule out any issues caused by cached data.
5. Checked for Mixed Content: I verified that all resources on my site are loaded over HTTPS.
6. Custom Domain Configuration: I ensured that my custom domain (if applicable) is properly configured with GitHub Pages and that DNS settings point to GitHub's servers.

Despite taking these measures, the security warning persists. I would appreciate any guidance or assistance you can provide to help resolve this issue.

**Comment by GabrielAlmeidag** [comment, marked as answer]
@Debanjan110d 👋😺

Hi there,

It’s great to see that you’ve already taken several steps to address the security warning on your GitHub Pages site. Despite your efforts, if the issue still persists, there are a few additional things you might want to consider.

First, it could be useful to check your site’s content against common blocklists. Sometimes, certain domains or external scripts can be flagged, triggering security warnings. Tools like Google Safe Browsing can help you see if your domain has been marked as unsafe.

Another aspect worth reviewing is the SSL certificate. Even if your site is set to use HTTPS, ensure that the SSL certificate is valid and correctly configured. If you’re using a custom domain, double-check that the SSL certificate was properly issued and is functioning as expected.

If you have a custom domain set up, revisiting your DNS settings and the CNAME file in your GitHub Pages configuration might help. Issues in these areas can sometimes cause conflicts that lead to security alerts.

Additionally, consider reverting your site to a previous version that worked fine if the problem is recent. This can help you identify whether a specific change triggered the warning.

You might also want to look at the security logs in your browser. By opening the console (using F12) while accessing your site, you can get more detailed information on what might be causing the alert.

If these steps still don’t resolve the issue, you might find it worthwhile to switch to a different platform, such as [Vercel](https://vercel.com/). Vercel integrates directly with GitHub and can make deployment and configuration much easier. Personally, I prefer using Vercel for my projects because of its ease of use and reliability.

I hope these suggestions help you resolve the issue. If you need further assistance, feel free to reach out!

**Reply by Debanjan110d** [reply]
Thanks a lot for the suggestions.
But I have alreay done most of the things its still not solving
do you know how can I reach out to the github support team via mail

**Reply by GabrielAlmeidag** [reply]
The email address for GitHub support is [support@github.com](mailto:support@github.com). You can also access GitHub support through their [support site](https://support.github.com/).  and they usually respond quickly via email.

Even if nothing seems to work, Vercel remains a great option for hosting your pages.

**Reply by Debanjan110d** [reply]
Thanks a lot
